src/EA.Iws.DataAccess/Repositories/Reports/ExportMovementsRepository.cs

```diff
@@ -19,58 +20,98 @@ public ExportMovementsRepository(IwsContext context)
             this.context = context;
         }
 
-        public async Task<ExportMovementsData> Get(DateTime from, DateTime to, UKCompetentAuthority competentAuthority)
+        public async Task<ExportMovementsData> Get(DateTime from, DateTime to, UKCompetentAuthority competentAuthority, OrganisationFilterOptions? organisationFilter, string organisationName)
         {
-            var movementData = await context.Database.SqlQuery<ExportMovementsData>(
-                @"WITH 
-                movementcreateddata AS (
+            string type = string.Empty;
+
+            if (organisationFilter == OrganisationFilterOptions.Notifier)
```

Comment:
I'd just change the values in the enum `OrganisationFilterOptions` to `Exporter` and `Importer` but keeping the display name as they are then you won't have to do this if and else check.